zwave_js_server/model/controller/__init__.py

```diff
@@ -646,9 +648,12 @@ def receive_event(self, event: Event) -> None:
                 f"Controller doesn't know how to handle/forward this event: {event.data}"
             )
 
-        validate_event_data(
-            event.data, "controller", event.type, CONTROLLER_EVENT_MODEL_MAP
-        )
+        if event.type not in CONTROLLER_EVENT_MODEL_MAP:
```

Comment:
I think we can remove this check and let a KeyError happen if the event type is missing.